packages/select/src/Select.js

```diff
@@ -26,6 +27,15 @@ const components = {
       <span className="sr-only">Clear all selections</span>
     </ClearIndicator>
   ),
+  Option: (props) => {
+    const innerProps = {
+      ...props.innerProps,
+      role: 'option',
+      'aria-selected': props.isSelected,
```

Comment:
I think we should add some unit tests to check for the existence of these props and confirm accessibility of the options

Reply:
@chrishavekost @jordan-a-young 
Are Adrian's most recent commits sufficient?